pyanaconda/ui/gui/spokes/storage.py

```diff
@@ -325,65 +327,54 @@ def __init__(self, *args, **kwargs):
-        if self._blivet_gui_radio_button.get_active():
-            return PARTITIONING_METHOD_BLIVET
-
-        return PARTITIONING_METHOD_AUTOMATIC
+        if self._auto_part_radio_button.get_active():
+            return PARTITIONING_METHOD_AUTOMATIC
+        else:
+            return PARTITIONING_METHOD_UNKNOWN_MANUAL
 
     def _method_radio_button_toggled(self, radio_button):
         """Triggered when one of the partitioning method radio buttons is toggled."""
         # Run only for an active radio button.
         if not radio_button.get_active():
             return
 
-        # Hide the encryption checkbox for Blivet GUI storage configuration,
-        # as Blivet GUI handles encryption per encrypted device, not globally.
-        # Hide it also for the interactive partitioning as CustomPartitioningSpoke
-        # provides support for encryption of mount points.
-        if self._get_selected_partitioning_method() != PARTITIONING_METHOD_AUTOMATIC:
-            self._encryption_revealer.set_reveal_child(False)
-            self._encrypted_checkbox.set_active(False)
+        # Show the encryption and reclaim space checkboxes only for automatic partitioning.
+        # Both custom interactive and blivet-gui include their own handling of encryption.
+        self._selected_partitioning_method = self._get_selected_partitioning_method()
+        if self._selected_partitioning_method == PARTITIONING_METHOD_AUTOMATIC:
+            self._auto_revealer.set_reveal_child(True)
         else:
-            self._encryption_revealer.set_reveal_child(True)
-
-        # Hide the reclaim space checkbox if automatic storage configuration is not used.
-        if self._get_selected_partitioning_method() == PARTITIONING_METHOD_AUTOMATIC:
-            self._reclaim_revealer.set_reveal_child(True)
-        else:
-            self._reclaim_revealer.set_reveal_child(False)
+            self._auto_revealer.set_reveal_child(False)
+            self._encrypted_checkbox.set_active(False)
+            self._reclaim_checkbox.set_active(False)
 
         # is this a change from the last used method ?
-        method_changed = self._get_selected_partitioning_method() != self._last_partitioning_method
+        method_changed = self._selected_partitioning_method != self._last_partitioning_method
         # are there any actions planned ?
         actions_planned = self.storage.devicetree.actions.find()
         if actions_planned:
             if method_changed:
                 # clear any existing messages from the info bar
                 # - this generally means various storage related error warnings
                 self.clear_info()
-                self.set_warning(_("Partitioning method changed - planned storage configuration "
-                                   "changes will be cancelled."))
+                #self.set_warning(_("Partitioning method changed - planned storage configuration "
+                #                   "changes will be cancelled."))
```

Comment:
Please remove this or add fixme but don't leave commented code in the code-base. For that is git history.